src/views/Generator/Generator.tsx

```diff
@@ -143,9 +144,14 @@ export function Generator() {
     <View
       title="Generator"
       subTitle={
-        <>
-          {getFileNameWithoutExtension(fileName)} {isDirty && '*'}
-        </>
+        <FileNameHeader
```

Comment:
This also fixes a recent regression that made it impossible to see the full file name if it overflows